docs/source/jupyter.rst

```diff
@@ -1,8 +1,15 @@
 Jupyter notebooks
 =================
 
-.. toctree::
-   :maxdepth: 1
+The following page contains a collection of practical examples and problem
+solutions solved with **poliastro**. From interplanetary transfers, to asteroid
+catching missions or even propagation under external perturbations. In addition,
+user might find other kind of tutorials related on how to use and customize
+plotting utilities, API capabilities and many other orbital mechanics topics.
+
+
+.. nbgallery::
+   :caption: Examples gallery
```

Comment:
I have a little problem here: the directory is called `examples`, the file is called `jupyter`, the page `Jupyter notebooks`, the TOC is `Tutorials` (after #971), and here you call them `Examples`. Do you think we can unify all this terminology? I think "Examples" everywhere is good enough. People reading the docs online don't necessarily care that they are Jupyter notebooks.

On the other hand, using @evildmp (hi! :blush:) [classification for documentation](https://documentation.divio.com/), these would be How-To Guides. But perhaps "examples" is good enough to keep. Your call :)

Reply:
Sure, I got the same feeling about this `example/tutorial/notebooks namespace`. **In fact, the caption after the `.. nbsphinx-gallery::` is not necessary at all, so we can simply get rid of it.**

Regarding previously shared talk, it looks very interesting. We could also adapt the new `Install, use and contribute` to `How to` section, maybe for another pull?

Reply:
Sure thing! Let's open an issue and revisit the classification later.

Reply:
Sounds good! :rocket:

Reply:
Done: https://github.com/poliastro/poliastro/issues/975